### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by brankyy** [idevelop/predator-vision#3]
Hi,
this heatmap works only by clicking the link on your homepage.
It will not work by typing the url manually or  on local.

**Comment by brankyy** [comment]
refreshing the page will also cause it out of order

**Comment by idevelop** [comment]
@brankyy thanks for flagging! Looks like a recent (?) security check added in Chrome:
```
DOMException: play() failed because the user didn't interact with the document first. https://goo.gl/xX8pDD
```

**Comment by idevelop** [comment]
I'll need to add a "Start" button that triggers the capture flow.

**Comment by idevelop** [comment]
Fixed in 2b23572b2cd9b0c571d19070852556af609ef539
